index.html

```diff
@@ -61,5 +61,194 @@ <h1 class="hero-intro">I’m Akorede. <br> Glad to see <br class="see-break"> yo
             </ul>
         </div>
     </header>
+    <main>
+        <section class="project-section">
+            <article class="project-container">
+                <div>
+                    <a href="">
+                        <img src="images/tonic.png" alt="">
+                    </a>
+                </div>
+                <div class="project-description">
+                    <h2>Tonic</h2>
+                    <ul class="experience-list">
+                        <li>CANOPY</li>
+                        <li>Backend Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.
+                    </p>
+                    <ul class="skill-list">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button>
+                        <a href="#">See Project</a>
+                    </button>
+                </div>
+            </article>
+            <article  class="project-container">
+                <div>
+                    <a href="">
+                        <img src="images/multi-stories.png" alt="">
+                    </a>
+                </div>
+                <div class="project-description">
+                    <h2>Multi-Post Stories</h2>
+                    <ul class="experience-list">
+                        <li>FACEBOOK</li>
+                        <li>Full Stack Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        Experimental content creation feature that allows users to 
+                        add to an existing story 
+                        over the course of a day without spamming their friends.
+                    </p>
+                    <ul class="skill-list">
+                        <li>HTML</li>
+                        <li>Ruby on rails</li>
+                        <li>CSS</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button>
+                        <a href="#">See Project</a>
+                    </button>
+                </div>
+            </article>
```

Comment:
Great work done so far 👍🏼   , only small issues to fix in your cards :  

1. please fix technologies list items  padding and margin 
2. the button should have these appearence only when pressed 
3. and the first list after card title the selected item should only be bolded  dont increase it's size 

![image (5)](https://github.com/Akorede3133/portfolio/assets/59707859/98c2210a-919a-4000-9a42-165e1eb027d9)

![image (7)](https://github.com/Akorede3133/portfolio/assets/59707859/9ce506b3-2199-4c79-a6f7-1de103eb10e0)